src/commands.rs

```diff
@@ -1811,14 +1811,25 @@ fn cmd_untrack(
     let wc_tree_id = locked_working_copy.write_tree();
     if wc_tree_id != new_tree_id {
         let wc_tree = store.get_tree(&RepoPath::root(), &wc_tree_id)?;
-        if let Some((path, _value)) = wc_tree.entries_matching(matcher.as_ref()).next() {
+        let added_back = wc_tree.entries_matching(matcher.as_ref()).collect_vec();
+        if !added_back.is_empty() {
             locked_working_copy.discard();
+            let path = added_back[0].0.clone();
```

Comment:
Why does `path` have to be cloned here?

Reply:
Good point, it doesn't need to be.  Fixed.